Topics/Teamwork.md

```diff
@@ -86,31 +86,31 @@ However, if the team member continues to be non-responsive and unmotivated despi
 
 **Be open to feedback, and seek it out**
 
-Although we just discussed how different being an effective leader can look from case to case, a staple in almost every effective leader is their openness to feedback and reiteration. As said in a leadership principle from a [famous company](https://www.amazon.jobs/content/en/our-workplace/leadership-principles), “Leaders do not believe their or their team’s body odor smells of perfume”. Leaders should certainly be prepared to give feedback to the people they work with, but it is just as important to be open to any feedback that comes back. After all, we just discussed how hard it can be to pin down what being an “effective leader” looks like in each situation, so why not let the people you lead help you? In fact, we believe that not only should you be open to feedback, but it is something that is worth actively seeking out as it is a leader’s responsibility to strive for constant improvement of the team, but also themselves and any mechanisms that they employ.
+Although we just discussed how different being an effective leader can look from case to case, a staple in almost every effective leader is their openness to feedback and reiteration. As said in a leadership principle from a [famous company](https://www.amazon.jobs/content/en/our-workplace/leadership-principles), “Leaders do not believe their or their team’s body odor smells of perfume”. This means that leaders should not view themselves or their team as perfect and above the need for feedback (smelling like perfume in the example), but should instead stay grounded and open to improvement. Leaders should certainly be prepared to give feedback to the people they work with, but it is just as important to be open to any feedback that comes back. After all, we just discussed how hard it can be to pin down what an “effective leader” looks like in each situation, so why not let the people you lead help you? In fact, we believe that not only should you be open to feedback, but it is something that is worth actively seeking out as it is a leader’s responsibility to not only strive for constant team improvement, but also improvement of themselves and any mechanisms that they employ. One simple way to seek out feedback is to directly ask the people you are working with for it on a regular basis (such as during your weekly check-ins if you have them), but it is important to do so respectfully and in a manner such that the other person feels comfortable speaking their mind.
```

Comment:
I think the sentence flows better, but I'll point out that you used "but" twice. See here: "but it is something that is worth actively seeking out as it is a leader’s responsibility to not only strive for constant team improvement, but also improvement of themselves and any mechanisms that they employ."

Reply:
Thanks for pointing this out, I thought about this myself as well haha. I agree it is a little weird, but should still be clear enough to the reader so I will leave it in. I like to think of the second "but" as a leaf node extending from the first "but" ;).